### PR TITLE
Remove duplicate package.json, update readme.

### LINKED ISSUE #30: Extra package.json?
I think that the second `package.json` is exteraneous, the one at `/{{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/package.json`. 

It's being read from `setup.py` but I think that `setup.py` should read the `/{{cookiecutter.project_shortname}}/package.json` instead, no?

### LINKED ISSUE #31: Update pytest-dash README usage
- Need to have the chrome driver on the path (or any other driver)
- Change the run example to `pytests tests --driver Chrome`.

### PR DESCRIPTION
**Pull request by T4rk1n**
- Remove the duplicate `package.json` and it's reference.
- Add root README to manifest.
- Replace duplicate `package.json` references with root.
- Update pytest usage info in generated readme.

Closes #30, Close #31

### CODE REVIEW

File: {{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     _sys.exit(1)
 
 _basepath = _os.path.dirname(__file__)
-_filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
+_filepath = _os.path.abspath(_os.path.join(_basepath, '..','package.json'))
```
Comment (nicolaskruchten) [inline {{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py]: Hmm will this actually work once the package is installed via `pip`?

Comment (nicolaskruchten) [reply {{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py]: Maybe this is why that `copyfile` was there?

File: {{cookiecutter.project_shortname}}/setup.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
 import json
-import os
 from setuptools import setup
 
 
-with open(os.path.join('{{cookiecutter.project_shortname}}', 'package.json')) as f:
+with open('package.json') as f:
```
Comment (nicolaskruchten) [inline {{cookiecutter.project_shortname}}/setup.py]: 👍

File: {{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py (hunk 1 above)
Comment (T4rk1n) [reply {{cookiecutter.project_shortname}}/{{cookiecutter.project_shortname}}/__init__.py]: I'll add a test that will package the build and run an app with it installed as a tarball.

File: {{cookiecutter.project_shortname}}/package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "prepublish": "npm run validate-init",
     "build:js-dev": "webpack --mode development",
     "build:js": "webpack --mode production",
-    "build:py": "node ./extract-meta.js src/lib/components > {{cookiecutter.project_shortname}}/metadata.json && copyfiles package.json {{cookiecutter.project_shortname}} && %(python_path) -c \"import dash; dash.development.component_loader.generate_classes('{{cookiecutter.project_shortname}}', '{{cookiecutter.project_shortname}}/metadata.json')\"",
+    "build:py": "node ./extract-meta.js src/lib/components > {{cookiecutter.project_shortname}}/metadata.json && %(python_path) -c \"import dash; dash.development.component_loader.generate_classes('{{cookiecutter.project_shortname}}', '{{cookiecutter.project_shortname}}/metadata.json')\"",
```
Comment (nicolaskruchten) [inline {{cookiecutter.project_shortname}}/package.json]: see my comment in https://github.com/plotly/dash/pull/451

**Comment by nicolaskruchten** [comment]
In principle this is mergable once we clear up the need for `package.json` being copied. Perhaps we should update this PR once https://github.com/plotly/dash/pull/451 is merged and have this PR encompass the new `dash-generate-components` call pattern?

**Comment by nicolaskruchten** [comment]
@T4rk1n can we address point 3 of #38 (renaming the copied `package.json`) in this PR ?

**Comment by T4rk1n** [comment]
Closing as outdated.

**Pull request closed without merging**